ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.13__amd64-nightly.yaml

```diff
@@ -1422,8 +1422,8 @@ tests:
     test:
     - chain: cucushift-sdn-migration-ovn
     workflow: cucushift-installer-rehearse-gcp-ipi-sdn
-- as: gcp-ipi-secureboot-confidential-computing-mini-perm-f60
-  cron: 28 3 23 2,4,6,8,10,12 *
+- as: gcp-ipi-confidential-secureboot-mini-perm-f60
```

Comment:
Let us move this job to a place where is close to `gcp-ipi-confidential-fips-mini-perm-f60-destructive`